packages/mds-types/transformers.ts

```diff
@@ -0,0 +1,87 @@
+import { UUID, Timestamp, VEHICLE_EVENT, Telemetry, VEHICLE_REASON } from './index'
+
+export interface VehicleEvent_v0_4_1 {
+  device_id: UUID
+  provider_id: UUID
+  timestamp: Timestamp
+  timestamp_long?: string | null
+  delta?: Timestamp | null
+  event_type: VEHICLE_EVENT
+  event_type_reason?: VEHICLE_REASON | null
+  telemetry_timestamp?: Timestamp | null
+  telemetry?: Telemetry | null
+  trip_id?: UUID | null
+  service_area_id?: UUID | null
+  recorded: Timestamp
+}
+
+export interface VehicleEvent_v1_0_0 {
+  device_id: UUID
+  provider_id: UUID
+  timestamp: Timestamp
+  timestamp_long?: string | null
+  delta?: Timestamp | null
+  vehicle_state: VEHICLE_EVENT
+  event_types: VEHICLE_REASON[]
```

Comment:
I think this type is incorrect, it should be something along the lines of:

`vehicle_state: 'available' | 'reserved' | 'on_trip' | 'elsewhere' | 'non_operational' | 'removed' | 'unknown'`
(I'm not going to enumerate out all of the new `event_types`, but they are certainly not what used to be `VEHICLE_REASON`s in 0.4.1 Agency

Reply:
Thanks for the review! I'll look at the types more closely and get back to you.